### ISSUE TITLE
Wrong generated declaration file for el-submenu

### ISSUE THREAD

**Issue by acupofspirt** [element-plus/element-plus#706]
_lib/el-submenu/index.d.ts_
```typescript
import Subel-menu from '../menu/src/submenu.vue';
export default Submenu;
```

`Subel-menu` is invalid identifier in TypeScript

**Comment by JeremyWuuuuu** [comment]
Looks @zazzaz did get all cases covered. Will be fixed soon😂

**Comment by JeremyWuuuuu** [comment]
Fixed in #709 , I am closing this, a new version should be rolled out.

**Comment by zazzaz** [comment]
fixed in 1.0.1-alpha.23
